interview-questions.md

```diff
@@ -4,49 +4,85 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. Why would you use the method super()?
 
-  Your answer:
+  Your answer: Super is used in the constructor of a child class to inherit the same properties as the constructor in the parent class.
 
-  Researched answer:
+  Researched answer: Within in the constructor method we call super() which passes the attributes from the constructor in the parent class (https://github.com/LEARNAcademy/Syllabus/blob/master/javascript/class-inheritance.md)
 
 
 
 2. What is a virtual DOM?
 
-  Your answer:
+  Your answer: DOM stands for Document Object Model.
```

Comment:
Great! However in an interview setting, I would give more of an answer to what a 'virtual' DOM is, rather than just defining what DOM stands for. I'm sure at this point this may have been a bit iffy to answer before having researched, so no worries!